pyproject.toml

```diff
@@ -12,21 +12,15 @@ readme = "README.md"
 
 [tool.poetry.dependencies]
 python = "^3.8"
-click = "*"
-importlib-metadata = ">=4.8.0"
-pyparsing = "^3.0.9"
-bioregistry = ">=0.9.43"
-deprecation = ">=2.1.0"
-linkml-runtime = ">=1.5.3"
+linkml-runtime = ">=1.5.5"
+importlib-metadata = ">=6.8.0"
+pansql = ">=0.0.1"
+sssom-schema = ">=0.14.0"
+bioregistry = ">=0.9.87"
```

Comment:
Id this needed?

Reply:
These are the same dependencies that existed before, just the order has changed.